tests/api_test.py

```diff
@@ -1,4 +1,7 @@
+import httpx
 import pytest
+import respx
```

Comment:
Please remove unused imports
```suggestion
import pytest
```